index.js

```diff
@@ -60,6 +61,60 @@ app.get('/category', (req, res) => {
 
 app.use('/event', eventHandler);
 
+app.post('/organizer',(req,res)=> {
+    let poc_name = req.body.poc_name;
+    let poc_contact= req.body.poc_contact;
+    let poc_email = req.body.poc_email;
+    let comittee= req.body.comittee;
+
+    let message="";
+
+    if((poc_name != "") && (comittee != "") && (poc_email != "") && (poc_contact != null)){
+        res.code=200;
+        message="success";
+    }
+    
+    else{
+        res.code=400;
+        message="incomplete";
+    }
+
+    if(res.code ==200){
+        
+        let organizer = db.collection("Organizer");
+        organizer.add ({poc_name: poc_name , comittee:comittee,poc_contact:poc_contact,poc_email:poc_email});
+    }
+    res.send(message);
+   
+}) 
+
+app.post('/category',(req,res)=> {
+    let name = req.body.name;
+    let sponser= req.body.sponser;
+    let audience_size = req.body.audience_size;
+    let duration= req.body.duration;
+
+    let message="";
+
+    if((name != "") && (sponser != null ) && (audience_size != "") && (duration != null)){
```

Comment:
again here sponser and duration is compared with null and other fields are compared with empty string